Tests fail on OS X, Node v0.10.22

Any ideas?

```
    ok 979 should be equal
    ok 980 should be equal
    ok 981 should be equal
    ok 982 should be equal
    ok 983 should be equal
    ok 984 should be equal
    ok 985 should be equal
    ok 986 should be equal
    ok 987 should be equal
    not ok 988 test/buffer.js
      ---
        exit:     143
        timedOut: true
        stderr:   |
          (node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit.
          Trace
              at EventEmitter.addListener (events.js:160:15)
              at test (/Users/feross/code/buffer-browserify/node_modules/tape/index.js:103:11)
              at Object.<anonymous> (/Users/feross/code/buffer-browserify/test/buffer.js:104:1)
              at Module._compile (module.js:456:26)
              at Object.Module._extensions..js (module.js:474:10)
              at Module.load (module.js:356:32)
              at Function.Module._load (module.js:312:12)
              at Function.Module.runMain (module.js:497:10)
              at startup (node.js:119:16)
              at node.js:901:3
          (node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit.
          Trace
              at EventEmitter.addListener (events.js:160:15)
              at test (/Users/feross/code/buffer-browserify/node_modules/tape/index.js:108:11)
              at Object.<anonymous> (/Users/feross/code/buffer-browserify/test/buffer.js:104:1)
              at Module._compile (module.js:456:26)
              at Object.Module._extensions..js (module.js:474:10)
              at Module.load (module.js:356:32)
              at Function.Module._load (module.js:312:12)
              at Function.Module.runMain (module.js:497:10)
              at startup (node.js:119:16)
              at node.js:901:3
          (node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit.
          Trace
              at EventEmitter.addListener (events.js:160:15)
              at test (/Users/feross/code/buffer-browserify/node_modules/tape/index.js:110:11)
              at Object.<anonymous> (/Users/feross/code/buffer-browserify/test/buffer.js:104:1)
              at Module._compile (module.js:456:26)
              at Object.Module._extensions..js (module.js:474:10)
              at Module.load (module.js:356:32)
              at Function.Module._load (module.js:312:12)
              at Function.Module.runMain (module.js:497:10)
              at startup (node.js:119:16)
              at node.js:901:3
        command:  "node" "buffer.js"
      ...

    1..988
    # tests 988
    # pass  987
    # fail  1

ok test/indexes.js .................................... 22/22
ok test/is-encoding.js .................................. 4/4
total ............................................. 1013/1014

not ok
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0
```

--- a/package.json
+++ b/package.json
@@ -11,8 +11,8 @@
         "base64-js" : "0.0.2"
     },
     "devDependencies" : {
-        "tape" : "~0.3.0",
-        "tap" : "~0.4.0"
+        "tape" : "~2.1.0",
+        "tap" : "~0.4.4"
     },
     "repository" : {
         "type" : "git",

